lib/Bio/EnsEMBL/DataCheck/Checks/CheckGenomicAlignTreeTable.pm

```diff
@@ -105,6 +105,30 @@ sub tests {
 
   }
 
+  my $sql_4 = qq/
+    SELECT COUNT(*)
+      FROM genomic_align_tree gat
+        JOIN genomic_align ga USING (node_id)
+        JOIN dnafrag df USING (dnafrag_id)
+      WHERE gat.right_index-gat.left_index!=1
+        AND df.coord_system_name!='ancestralsegment'
```

Comment:
```suggestion
        AND df.coord_system_name != 'ancestralsegment'
```
More breathing space for the eyes, again a potentially ignore-able formatting suggestion